torchvision/datasets/__init__.py

```diff
@@ -128,6 +129,7 @@
     "InStereo2k",
     "ETH3DStereo",
     "wrap_dataset_for_transforms_v2",
+    "Imagenette",
```

Comment:
We'll need it in `datasets.rst` as well for the docs (right next to the `ImagNet` entry?)